cypress/support/commands.js

```diff
@@ -23,3 +24,34 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', (user = {
+  email: Cypress.env('email'),
+  password: Cypress.env('password')
+}) => {
+  cy.visit('/login');
+  cy.get('#email').should('be.visible').type(user.email)
+  cy.get('#password').should('be.visible').type(user.password)
+  cy.get('button[type="submit"]').should('be.visible').click()
+  cy.url().should('eq', 'https://notes-serverless-app.com/')  
```

Comment:
If I pass invalid credentials, this step will always fail. Maybe the expectation should go in the test, not the app action.

Reply:
Done!